File: java/49 SumAverage2dArray.java
package code;
import java.util.*;
public class Test {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		int arr1[][]= {{1,2,3,2,1},{4,5,6,4},{7,8,9}};
		int sum=0;
		int counter=0;
		boolean found=false;
		for(int i=0;i<arr1.length;i++) {
			for(int j=0;j<arr1[i].length;j++) {
				sum+=arr1[i][j];
				counter++;
			}
		}
		System.out.println("Sum is "+sum+" and average is "+sum/counter);
	}
}

File: python/49 SumAverage2dArray.py
arr=[[1,2,3,2,1],[4,5,6,5],[7,8,9]]
sum=0
counter=0
found=False
for i in range(len(arr)):
    for j in range(len(arr[i])):
        sum+=arr[i][j]
        counter+=1
print(f"Sum is {sum} and average is {sum/counter}")